block fields are private

I can't get at the parsed `block_type` or `data` field of the `Block` struct because they are private.

```
error[E0616]: field `block_type` of struct `nom_pem::Block` is private
  --> src/main.rs:21:22
   |
21 |     println!("{:?}", block.block_type);
   |                      ^^^^^^^^^^^^^^^^
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -52,9 +52,9 @@ mod test;
 /// structure representing one PEM block
 #[derive(Debug)]
 pub struct Block<'a> {
-    block_type: &'a str,
-    headers: Vec<HeaderEntry<'a>>,
-    data: Vec<u8>
+    pub block_type: &'a str,
+    pub headers: Vec<HeaderEntry<'a>>,
+    pub data: Vec<u8>
 }
 
 #[derive(Debug)]
@@ -88,4 +88,3 @@ impl<'a> fmt::Display for Block<'a> {
         write!(f, "-----END {}-----\n", &self.block_type)
     }
 }
-

